python/pyspark/ml/feature.py

```diff
@@ -1308,6 +1357,20 @@ def setMissingValue(self, value):
         """
         return self._set(missingValue=value)
 
+    @since("3.0.0")
+    def setInputCol(self, value):
+        """
+        Sets the value of :py:attr:`inputCol`.
+        """
+        return self._set(inputCol=value)
+
+    @since("3.0.0")
+    def setOutputCol(self, value):
+        """
+        Sets the value of :py:attr:`outputCol`.
+        """
+        return self._set(outputCol=value)
+
```

Comment:
I added the above setters explicitly because I need to add these later on anyways.